uber/automated_emails.py

```diff
@@ -31,6 +31,12 @@ def __init__(self, model, subject, template, filter, *, sender=None, extra_data=
     def __repr__(self):
         return '<{}: {!r}>'.format(self.__class__.__name__, self.subject)
 
+    @property
+    def approved(self):
+        """Returns a boolean indicating whether this email has been approved."""
+        with Session() as session:
+            return bool(session.query(ApprovedEmail).filter_by(ident=self.ident).first())
```

Comment:
This also seemed like a useful feature.  Strictly speaking it's not necessary, but it made implementing some new config properties a lot easier:
https://github.com/magfest/panels/pull/52/files#diff-b41827efa8a76a13ef812b9c1d774b6cR16